src/api/tests/common.py

```diff
@@ -49,3 +49,17 @@ def less_console_noise():
             handler.setStream(restore[handler.name])
         # close the file we opened
         devnull.close()
+
+
+def less_console_noise_decorator(func):
```

Comment:
Very cool. This could be how we refactor all our tests and do all our future tests @abroddrick @dave-kennedy-ecs